test/performance/shmem_perf_suite/shmem_bibw_atomics_perf.c

```diff
@@ -152,7 +170,6 @@ static inline void bw_set_metric_info_len(perf_metrics_t *metric_info)
     int snode = streaming_node(*metric_info);
     atomic_op_type op_type = OP_ADD;
     metric_info->b_type = BI_DIR;
```

Comment:
In case github lost my other comment.  Should this like be moved up to before the call to `streaming_node`?